### PR TITLE
MAWB-3 (add readme file)

### PR DESCRIPTION
**Pull request by YatKat**

### CODE REVIEW

**Comment by YatKat** [comment]
I pushed this commit to my origin MAWB-3 and then opened pull request in bchupika/master. It's the right way to create Pull requests! I've made screenshots while doing it, so i'll make next commit and add screenshots how to do it in details.

**Comment by YatKat** [comment]
You can see how it looks like in README.md format on GitHub, you can go to my YatKat GitHub account and see <MAWB-3/code>. Or go by this link: https://github.com/YatKat/mate_academy_wikibot_NOV18/blob/MAWB-3/README.md

File: README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Add reviwers to your pull request.
+
+After the PR test completed successfully and your changes are reviewed ‘Rebase and Merge’ the pull request.
+
+If you have multiple commits and want to squash them into a single commit then either do it locally or click ‘Squash and merge’ when merging the PR.
+
+# <a name="statistic"></a>Statistic of code quality
+
+[![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/boroda4436/mate_academy_wikibot_NOV18.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/boroda4436/mate_academy_wikibot_NOV18/context:java)
```
Comment (boroda4436) [inline README.md]: let's move it to the top of the readme. Just a badge. 
See example here: https://github.com/spring-projects/spring-security/blob/master/README.adoc

**Comment by boroda4436** [comment]
Realy cool README! I like it)

File: README.md (hunk 1 above)
Comment (YatKat) [reply README.md]: Ok. I'll wait for some more comments not to do a lot of small commits. And then I'll make all changes in whole.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+* CircleCI for checkstyle and build check
+* Travis CI for checkstyle and build check
+* HttpClient 4.5.6
+* Jackson
+
+<hr>
+
+# <a name="user-start"></a>For user. How to start?
+
+Install Git
```
Comment (boroda4436) [inline README.md]: I would recommend adding here the information about how to use Bot (for example on a web browser or mobile phone). Let's move the _install git_ section to dev

Comment (YatKat) [reply README.md]: +

**Comment by boroda4436** [comment]
@YatKat please update branch and merge the pr

**Pull request merged**